### PR TITLE
Fixed Perfect Dark textures on Nvidia GPUs

### PR DESCRIPTION
**Pull request by AdrianCassar**

### CODE REVIEW

File: patches/584109C2 - Perfect Dark (Dec 3, 2009 Prototype).patch.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+title_name = "Perfect Dark (Dec 3, 2009 Prototype)"
+title_id = "584109C2"
+hash = "EC379B5CAE85D8B6" # default.xex
+#media_id = "00000000"
+
+[[patch]]
+    name = "Fix textures on Nvidia GPUs"
+    desc = "Fixed noise on textures with Nvidia GPUs"
```
Comment (Margen67) [inline patches/584109C2 - Perfect Dark (Dec 3, 2009 Prototype).patch.toml:8]:
```suggestion
    name = "Fix texture noise on NVIDIA GPUs"
```

File: patches/584109C2 - Perfect Dark.patch.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+title_name = "Perfect Dark"
+title_id = "584109C2"
+hash = "F11821D5051987EE" # default.xex
+#media_id = "00000000"
+
+[[patch]]
+    name = "Fix textures on Nvidia GPUs"
+    desc = "Fixed noise on textures with Nvidia GPUs"
```
Comment (Margen67) [inline patches/584109C2 - Perfect Dark.patch.toml:8]:
```suggestion
    name = "Fix texture noise on NVIDIA GPUs"
```

File: patches/584109C2 - Perfect Dark (Dec 3, 2009 Prototype).patch.toml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+title_name = "Perfect Dark (Dec 3, 2009 Prototype)"
+title_id = "584109C2"
```
Comment (Margen67) [inline patches/584109C2 - Perfect Dark (Dec 3, 2009 Prototype).patch.toml:2]:
```suggestion
title_id = "584109C2" # XA-2498
```

File: patches/584109C2 - Perfect Dark.patch.toml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+title_name = "Perfect Dark"
+title_id = "584109C2"
```
Comment (Margen67) [inline patches/584109C2 - Perfect Dark.patch.toml:2]:
```suggestion
title_id = "584109C2" # XA-2498
```

File: patches/584109C2 - Perfect Dark (Dec 3, 2009 Prototype).patch.toml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+hash = "EC379B5CAE85D8B6" # default.xex
+#media_id = "00000000"
```
Comment (Margen67) [inline patches/584109C2 - Perfect Dark (Dec 3, 2009 Prototype).patch.toml:4]: Is this the real media ID?

File: patches/584109C2 - Perfect Dark.patch.toml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+hash = "F11821D5051987EE" # default.xex
+#media_id = "00000000"
```
Comment (Margen67) [inline patches/584109C2 - Perfect Dark.patch.toml:4]: Is this the real media ID?

File: patches/584109C2 - Perfect Dark (Dec 3, 2009 Prototype).patch.toml (hunk 5 above)
Comment (AdrianCassar) [reply patches/584109C2 - Perfect Dark (Dec 3, 2009 Prototype).patch.toml:4]: yep, that's what it says in the log.

File: patches/584109C2 - Perfect Dark.patch.toml (hunk 6 above)
Comment (AdrianCassar) [reply patches/584109C2 - Perfect Dark.patch.toml:4]: yep, that's what it says in the log.

**Pull request merged**